serverless/packages/functions/src/model/delete_model.ts

```diff
@@ -0,0 +1,43 @@
+import { APIGatewayProxyEventV2 } from "aws-lambda";
+import { DynamoDBClient, DeleteItemCommand } from '@aws-sdk/client-dynamodb';
+
+export async function handler(event : APIGatewayProxyEventV2) {
+    let queryParams = null;
+    if (event && (queryParams = event['pathParameters']) != null) {
+        const model_id: string | undefined = queryParams['model_id'];
+        if (model_id == undefined) {
+            return {
+                statusCode: 401,
+                body: JSON.stringify({message: "Malformed request content - model ID missing."})

```

Comment:
does this error message structure exist for other dynamo endpoints for trainspace and user?

Reply:
yup, the error handling is standard across all of the trainspace, user, and model endpoints.